presto-docs/src/main/sphinx/sql/drop-function.rst

```diff
@@ -7,14 +7,16 @@ Synopsis
 
 .. code-block:: none
 
-    DROP FUNCTION [ IF EXISTS ] qualified_function_name [ ( parameter_type[, ...] ) ]
+    DROP [TEMPORARY] FUNCTION [ IF EXISTS ] qualified_function_name [ ( parameter_type[, ...] ) ]
 
 
 Description
 -----------
 
 Drop an existing function.
 
+The ``TEMPORARY`` modifier must be used to drop a temporary function.
+
```

Comment:
I think it should be descriptive, not prescriprive. Say: 

When temporary is specified, a temporary function with the given signature is dropped.

Reply:
done

Reply:
Should this change go into the first commit?

Reply:
done